docker/v1.25.0/ubuntu18.04/Dockerfile

```
@@ -0,0 +1,101 @@
+#
+#  Copyright (c) 2024, Intel Corporation
+#
+#  SPDX-License-Identifier: BSD-3-Clause
+
+
+FROM ubuntu:18.04
+LABEL maintainer="Arina Neshlyaeva <arina.neshlyaeva@intel.com>"
+SHELL ["/bin/bash", "-c"]
+
+ARG REPO=ispc/ispc
+ARG SHA=main
+ARG LTO=OFF
```

For release docker I'd update default arguments values.
```suggestion
ARG SHA=v1.25.0
ARG LTO=ON
```

---

Done